main.tf

```diff
@@ -274,15 +266,22 @@ resource "kubectl_manifest" "karpenter_node_pool_arm" {
     apiVersion: karpenter.sh/v1beta1
     kind: NodePool
     metadata:
-      name: default-arm
+      name: truemark-arm64
     spec:
+      disruption:
+        budgets:
+          - nodes: 10%
```

Comment:
Should this be more aggressive?

Reply:
we can change this later, for now I believe going with 10% is better option. We can expose this as variable if needed so that things can be controlled per deployment basis.